webapp-django/crashstats/crashstats/context_processors.py

```diff
@@ -3,6 +3,7 @@
 
 def settings(request):
     return {
+        'SETTINGS': django_settings,
         'DEBUG': django_settings.DEBUG,
         'GOOGLE_ANALYTICS_ID': django_settings.GOOGLE_ANALYTICS_ID,
```

Comment:
We could probably just add `SETTINGS` and then remove the other two since they're in the settings. Looks like `DEBUG` isn't used at all. The other one is used only once.